src/Cli/dotnet/commands/dotnet-workload/install/LocalizableStrings.resx

```diff
@@ -298,7 +298,7 @@
     <value>Workload updates are available. Run `dotnet workload list` for more information.</value>
   </data>
   <data name="PendingReboot" xml:space="preserve">
-    <value>The machine has a pending reboot. Installation will continue, but you may need to restart.</value>
```

Comment:
We used to say "Installation" will continue even in an uninstall comand, and now in the clean command...

Reply:
Good catch